lib/base-compiler.js

```diff
@@ -62,6 +62,9 @@ export class BaseCompiler {
 
         this.alwaysResetLdPath = this.env.ceProps('alwaysResetLdPath');
         this.delayCleanupTemp = this.env.ceProps('delayCleanupTemp', false);
+        this.stubRe = new RegExp(this.compilerProps('stubRe'));
```

Comment:
I wish could not assign to `this.compilerProps` at all? It's annoying the subclasses need it too. Maybe a better refactoring is to call the constructor with the `compilerProps`, then all the child classes can have it in their constructor params to not hvae to store it. Thinking out loud.